ARE/middleware/src/main/java/eu/asterics/mw/are/AsapiSupport.java

```diff
@@ -381,29 +341,9 @@ public Object call() throws Exception {
 
                         @Override
                         public List<String> call() throws Exception {
-                            logger.fine("\n\ngetBundelDescriptors\n\n");
+                            logger.fine("getBundelDescriptors()");
                             List<String> res = new ArrayList<String>();
-
-                            List<Bundle> bundleList = DeploymentManager.instance.getBundleManager()
-                                    .getInstallableBundleList();
-                            for (Bundle bundle : bundleList) {
-                                URL url = bundle.getResource(DefaultBundleModelParser.BUNDLE_DESCRIPTOR_RELATIVE_URI);
-                                if (url != null) {
-                                    try {
-                                        res.add(ResourceRegistry.getResourceContentAsString(url.openStream()));
-                                    } catch (IOException e) {
-                                        // TODO Auto-generated catch block
-                                        // throw (new
-                                        // AREAsapiException(e.getMessage()));
-                                        // keep it by just logging
-                                        // because we have to assume that
-                                        // several bundles are not supported for
-                                        // the platform
-                                        AstericsErrorHandling.instance.getLogger()
-                                                .warning("Could not get AsTeRICS bundle descriptor for url: " + url);
-                                    }
-                                }
-                            }                            
+                            res.add(getComponentDescriptorsAsXml());
```

Comment:
so whats the difference between this method `getBundelDescriptors()` and the other one `getComponentDescriptorsAsXml()` if you are returning the result of `getComponentDescriptorsAsXml()` here? Why not remove one of the two methods?

Reply:
by the way: `getBundelDescriptors()` has a typo, shouldnt it be `getBundleDescriptors()`? So maybe we can change it...

Reply:
unfortunately both methods are used, although the functionality is more a less equivalent. One is used by the C# ACS and the other one is used by the WebACS. I actually already did a hack by just adding the contents of the whole component collection to the first element of the List<String> collection in getBundleDescriptors.

Yes fixing the typo makes sense.